src/DatePicker/DatePicker.test.js

```diff
@@ -23,7 +23,7 @@ describe('<DatePicker />', () => {
     });
 
     test('open/close by calendar icon button', () => {
-        wrapper = mount(defaultDatePicker);
+        wrapper = mount(defaultDatePicker).children().children();
```

Comment:
There are ~80 instances of this `.children().children()` pattern—can we write a single mount function with a comment for why we need to dive down 2 levels?

Reply:
Definitely should be on the to do for the next iteration ✅